app/assets/javascripts/_page_show.es6

```diff
@@ -95,3 +95,14 @@
     $document.on('turbolinks:load', execPageBuilderFunctions);
 })(window.jQuery);
 
+jQuery(document).ready(function () {
+    var isChrome = /Chrome/.test(navigator.userAgent) && /Google Inc/.test(navigator.vendor);
+    if (window.location.hash && isChrome) {
+        setTimeout(function () {
+            var hash = window.location.hash;
+            window.location.hash = "";
+            window.location.hash = hash;
+        }, 300);
+    }
+});
+
```

Comment:
What does this code do? Removing it didn't seem to affect the desired outcome.

Reply:
This is for a work around in chrome...  yeah I wasn't expecting to have to add any js but found that the old school anchor links don't work in chrome without this js code in place to find the the anchor.  Were you using chrome or safari?

Reply:
I was using chrome, and it was working correctly.

Reply:
We can test without it.  This is what I was running in to.. https://stackoverflow.com/questions/51887886/anchor-links-not-working-on-chrome   but this may only affect certain versions of chrome.. or may be a chrome/windows thing.. not sure.  My version of chrome is 106.0.5249.61.